config/index.js

```diff
@@ -83,9 +84,7 @@ config.globals = {
   '__DEBUG__'    : config.env === 'development' && !argv.no_debug,
   '__COVERAGE__' : !argv.watch && config.env === 'test',
   '__BASENAME__' : JSON.stringify(process.env.BASENAME || ''),
-  '__CONTROLLER_API__' : config.env === 'test' ?
-    `${config.controller_service}/api/v1` :
-    JSON.stringify(`${config.controller_service}/api/v1`),
+  '__CONTROLLER_API__' : JSON.stringify(config.controller_service || ''),
```

Comment:
the removal of /api/v1 here means we will need to update the toolchain and the live pipeline. It currently only passes the host.domain part. Goal was to be able to have multiple API version and let the client - in this case the webui the choice to use v1 or v2, etc. If you want to stick to full host.domain/path make sure to update logistics-wizard-toolchain and the live pipeline.